attunement.lic

```diff
@@ -146,13 +149,23 @@ class Attunement
     waitrt?
   end
 
+  def magic_cleanup
+    return if @training_spells.empty?
```

Comment:
If this is the only use of `@training_spells`, then I suggest instead storing `@training_spells.empty?` once.

On second thought, why not make this conditional on `@magic_filler`?